lib/whitehall_importer/create_revision.rb

```diff
@@ -58,6 +57,14 @@ def call
 
   private
 
+    def translation
+      @translation ||= whitehall_edition["translations"].find do |t|
+        t["locale"] == document.locale
+      end
+
+      @translation || raise(AbortImportError, "Translation #{document.locale} missing")
```

Comment:
~~Is it worth writing a test for the abort error?~~ Ah I see it is added in the next commit. Could we have one commit for adding & testing the abort error?